integration-box/Kinesis/lambda_function.py

```diff
@@ -0,0 +1,35 @@
+import base64, time, json, sys
+import dateutil, dateutil.parser, datetime
+import urllib3
+
+td_database = '<YOUR_DATABASE_NAME>'
+td_table = '<YOUR_TABLE_NAME>'
+td_master_key = '<YOUR_WRITE_ONLY_API_KEY>'
+td_endpoint = 'https://in.treasuredata.com/js/v3/event'
+ 
+def upload_td(records):
+    headers = {
+        'Content-Type': 'application/json',
+        'X-TD-Data-Type': 'k',
+        'X-TD-Write-Key': td_master_key,
+    }
+    data = json.dumps({ '%s.%s' % (td_database, td_table): records }).encode("utf-8")
+    http = urllib3.PoolManager()
+    req = http.request('POST',url=td_endpoint, body=data, headers=headers)
+ 
+    resp = json.loads(req.data.decode('utf-8'))
+    print("Success: %s records" % len(resp))
+ 
+def lambda_handler(event):
```

Comment:
Just in case, what's the lambda runtime version? I tried this on Python 3.8 and ran into this issue..
errorMessage": "lambda_handler() takes 1 positional argument but 2 were given
I think `def lamdba_handler(event, context):` 
ref: https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html

Reply:
I have added context and did a test run on aws console you shared, seem to work.